packages/node_modules/@webex/internal-plugin-conversation/test/integration/spec/encryption.js

```diff
@@ -142,6 +143,33 @@ describe('plugin-conversation', () => {
           }));
       });
 
+      describe('#addReaction()', () => {
+        let activity;
+
+        beforeEach(() =>
+          webex.internal.conversation.post(conversation, {displayName: 'Ahoy'}).then((a) => {
+            activity = a;
+
+            // consume the mercury event for posting the comment
+            return Promise.resolve(expectEvent(10000, 'event:conversation.activity', checkov.webex.internal.mercury, 'checkov should get the comment event'));
```

Comment:
What is the purpose of this check? We'd prefer not to have expectations in the setup (beforeEach).